src/pljson_parser.impl.sql

```diff
@@ -22,25 +22,25 @@ create or replace package body pljson_parser as
   */
 
   decimalpoint varchar2(1 char) := '.';
+  -- Why 222? - I like it.
```

Comment:
This comment is not useful.

Reply:
I disagree. This random value between 200 - 500. with best performence

Reply:
Fixed